bin/thumbnail

```diff
@@ -551,6 +572,13 @@ function thumb() {
 
 		if eval ${magick}; then
 			echo ${magick}
+			if [[ $gif -eq 1 && $animated -eq 1 && $webp -eq 1 ]]; then
+			    WEBP_OUT=${OUT}.webp
+			    gif2webp_result="${TIMEOUT} ${TIMEOUT_TIME} ${GIF2WEBP} ${GIF2WEBP_OPTIONS} "${OUT}" -o "${WEBP_OUT}""
+			    if eval ${gif2webp_result}; then
+			        mv ${WEBP_OUT} ${OUT}
+			    fi
```

Comment:
After offline talk we decided to leave this and serve original GIFs and monitor failures if there are any. Then we decide if we should fail and return error code or not.